concepts/for-loops/about.md

```diff
@@ -0,0 +1,124 @@
+# About
+
+## General Syntax
+
+The for loop is one of the most commonly used statements to repeatedly execute some logic.
+In JavaScript it consists of the `for` keyword, a _header_ wrapped in round brackets and a code block that contains the _body_ of the loop wrapped in curly brackets.
+
+```javascript
+for (initialization; condition; step) {
+  // code that is executed repeatedly as long as the condition is true
+}
+```
+
+## Header
+
+The initialization usually sets up a counter variable, the condition checks whether the loop should be continued or stopped and the step increments the counter at the end of each repetition.
+The individual parts of the header are separated by semicolons.
+
+```javascript
+const list = ['a', 'b', 'c'];
+for (let i = 0; i < list.length; i++) {
+  // code that should be executed for each item in the array
+}
+
+// i is not available here
+```
+
+Variables that are defined in the header with `let` are scoped to the for statement.
+In the example above you would see `ReferenceError: i is not defined` if you would try to access the counter outside of the for loop.
```

Comment:
Since this is more related to variable scope with `let`, is there a concept for that which could be referenced as well?

Reply:
You are right, this is not really related to the for loop and should be part of the concept were we deep dive into variables. I removed it here.